blockbase/sass/ponyfill.scss

```diff
@@ -29,3 +29,6 @@
 @import "blocks/video";
 @import "blocks/columns";
 @import "post/meta";
+.wp-block-post-template {
```

Comment:
Did you mean to import `blocks/post-template` partial?

Reply:
oh yeah duh 🤦

Reply:
done in aa4e21f7e535497425abb6bb4bae192447a65973